README.md

```diff
@@ -66,17 +67,19 @@ recipe = [
     SmoothQuantModifier(smoothing_strength=0.8),
     GPTQModifier(scheme="W8A8", targets="Linear", ignore=["lm_head"]),
 ]
+model = AutoModelForCausalLM("TinyLlama/TinyLlama-1.1B-Chat-v1.0")
 
 # Apply quantization using the built in open_platypus dataset.
 #   * See examples for demos showing how to pass a custom calibration set
 oneshot(
-    model="TinyLlama/TinyLlama-1.1B-Chat-v1.0",
+    model=model,
     dataset="open_platypus",
     recipe=recipe,
-    output_dir="TinyLlama-1.1B-Chat-v1.0-INT8",
```

Comment:
Did we lose the ability to use output_dir? I use this pathway a lot since it avoided "re-compressing" the model when saving externally

Reply:
no it would save if one provides a non-default output_dir (not ./output).